lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -100,7 +103,8 @@ def strip_coderay_prefix(hash)
         private
         def modernize_coderay_config
           if highlighter == "coderay"
-            Jekyll::Deprecator.deprecation_message "You are using 'kramdown.coderay' in your configuration, " \
+            Jekyll::Deprecator.deprecation_message \
+              "You are using 'kramdown.coderay' in your configuration, " \
               "please use 'syntax_highlighter_opts' instead."
```

Comment:
This warning is 92 characters by itself 😳
Perhaps we could use a `\n` in there?